snabble/Android-SDK
# Add support for custom properties (e.g. external billing, subject length)

Custom properties can now be added and implemented easier. As an example the subject text edit for the external billing has been make configurable using a `CustomProperty`.

APPS-1491 _(APPS-1497)_

### How to test?

* Checkout this branch for an app making use of external billing, limit the subject length and check if it works.


### Definition of Done


#### App Tests
- ~~Minified getestet? _(aka. Release Build)_~~
- ~~Environements beachtet _(Production/Staging)_~~
- ~~Unterstützte Sprachen sind getestet~~
- ~~Light-/Dark-Mode getestet~~

#### Testing
- ~~Tests geschrieben _(aka Unit-Tests, Integration-Tests)_~~
